src/DGmethods/DGBalanceLawDiscretizations.jl

```diff
@@ -134,12 +139,35 @@ where
   set to zero is this desired)
 - `nM` is the unit outward normal to the face with respect to the minus side
   (`MVector` of length `3`)
-- `QM` and `QP` are the minus and plus side values (`MArray`)
+- `QM` and `QP` are the minus and plus side states (`MArray`)
 - `auxM` and `auxP` are the auxiliary states (`MArray`)
 - `t` is the current simulation time
 Warning: Modifications to `nM`, `QM`, `auxM`, `QP`, or `auxP` may have side
 effects and should not be done
 
+If `grid.topology` has a boundary then the function
+`inviscid_numerical_boundary_flux!` must be specified. This function is called
+with the data from the neighbouring DOF as
+```
+inviscid_numerical_boundary_flux!(F, nM, QM, auxM, QP, auxP, bctype, t)
+```
+where
+- `F` is an `MArray` of size `(dim, length_state_vector)` to be filled with the
+  numerical flux across the face (note that this is uninitialized so user must
+  set to zero is this desired)
+- `nM` is the unit outward normal to the face with respect to the minus side
+  (`MVector` of length `3`)
+- `QM` and `QP` are the minus and plus side states (`MArray`)
+- `auxM` and `auxP` are the auxiliary states (`MArray`)
+- `bctype` is the boundary condition flag for the connected face and element of
+   `grid.elemtobndy`
+- `t` is the current simulation time
+Note: `QP` and `auxP` are filled with values reflectes in `grid.vmapP`, and
+these values may be modified by the calling function.
+
+Warning: Modifications to `nM`, `QM`, or `auxM` side effects and should not be
```

Comment:
Should "to `nM`, `QM`, or `auxM` side effects" be "to `nM`, `QM`, or `auxM` cause side effects"?

Reply:
yes